fml/platform/fuchsia/message_loop_fuchsia.cc

```diff
@@ -32,12 +33,19 @@ void MessageLoopFuchsia::WakeUp(fml::TimePoint time_point) {
     due_time = zx::nsec((time_point - now).ToNanoseconds());
   }
 
-  if (task_.is_pending()) {
-    task_.Cancel();
+  task_mutex_.lock();
+
+  auto status = task_.Cancel();
+  FML_DCHECK(status == ZX_OK || status == ZX_ERR_NOT_FOUND);
+  if (status == ZX_ERR_NOT_FOUND) {
+    FML_LOG(INFO)
```

Comment:
Is this worth logging? The first task posted to the message loop would log here. It may be more worthwhile to log non OK or NOT found type error. But, you are already asserting for that. So it should be fine to just remove this.

Reply:
+1